npm/ng-packs/packages/permission-management/src/lib/components/permission-management.component.ts

```diff
@@ -137,8 +139,11 @@ export class PermissionManagementComponent
 
   isGrantedByOtherProviderName(grantedProviders: ProviderInfoDto[]): boolean {
     if (grantedProviders.length) {
+      this.disableSelectAllTab =
```

Comment:
`isGrantedByOtherProviderName` function is used in the if statement. And this assignment creates a side effect. Please create another method for this assignment